Helper/Data.php

```diff
@@ -87,6 +91,15 @@ public function getNewsletter()
         return $this->getScopeSetting(self::NEWSLETTER);
     }
 
+    public function getOptInSetting()
+    {
+        if($this->getScopeSetting(self::USING_KLAVIYO_LIST_OPT_IN)) {
+            return SUBSCRIBE;
+        } else {
```

Comment:
use an else if, i'd rather be explicit ?

Reply:
actually nvm, sorry this is good - to use an else because it's a bool.. just do the return self::*